### ISSUE TITLE
Improve sidebar accessibility on mobile screen

### ISSUE THREAD

**Issue by Perdolique** [Perdolique/perd#125]
Content under the opened sidebar can be marked as inert or so. Also, it should be possible to use native dialogues or popups (as an idea).

**Comment by Perdolique** [comment]
The mobile interface now uses bottom navigation. The sidebar is hidden below the desktop breakpoint and is not a mobile overlay. The old overlay focus and inert-content scenario no longer applies.
